pinecone/utils/error_handling.py

```diff
@@ -3,19 +3,14 @@
 
 from urllib3.exceptions import MaxRetryError, ProtocolError
 
-from pinecone import Config
-
 def validate_and_convert_errors(func):
     @wraps(func)
     def inner_func(*args, **kwargs):
-        Config.validate()  # raises exceptions in case of invalid config
```

Comment:
This happens within the init of the `Config` itself, right? Not really necessary to have it here.